### PR TITLE
refactor:  simplified index.tsx 

### PR DESCRIPTION
**Pull request by Simplyalex99**
Separated the contents in index.tsx file for home page. Index.tsx file contains the flex layout so that it can be adjusted while the content is seperated, and states still kept to the top level.

### CODE REVIEW

File: packages/web/src/pages/index.tsx [hunk 1]
```diff
@@ -60,104 +57,23 @@ const HomePage: React.FC<IndexPageProps> = ({ vars }) => {
               {currentUserLoading || (
                 <>
                   <Link href={`/user/${user?.username}`}>
-                    <a className="flex items-center space-x-4">
-                      <img
```
Comment (Simplyalex99) [inline packages/web/src/pages/index.tsx:64]: Opps, I forgot to remove this part when doing Pull request since UserSection components has this already.

**Review by BronzW — COMMENTED** [review]
I will check this out later, also there are some merge conflicts that needs to be fixed

**Comment by Simplyalex99** [comment]
No problem, I can close this PR, and submit a new one after resolving the conflicts if that's preferred.

**Comment by heybereket** [comment]
> No problem, I can close this PR, and submit a new one after resolving the conflicts if that's preferred.

No, keep it the same PR. Just fix the merge conflicts and commit those fixes to your branch.

**Pull request closed without merging**